tests/time_tests/scripts/run_timetest.py

```diff
@@ -49,11 +49,17 @@ def parse_stats(stats: list, res: dict):
                     parse_stats(v, res)
 
 
-def aggregate_stats(stats: dict):
+def aggregate_stats(stats: dict, args:dict):
     """Aggregate provided statistics"""
-    return {step_name: {"avg": statistics.mean(duration_list),
+    agg_stats = {step_name: {"avg": statistics.mean(duration_list),
                         "stdev": statistics.stdev(duration_list) if len(duration_list) > 1 else 0}
             for step_name, duration_list in stats.items()}
+    if args["model_cache"]: # FIL-First Inference Latency
+        agg_stats['time_to_first_inference_result']={'avg': (agg_stats['time_to_inference']['avg']+agg_stats['first_inference']['avg']) / 10 **3}
+    else: #FEIL-First Ever Inference Latency
+        agg_stats['time_to_first_inference_result']={'avg': (agg_stats['time_to_inference']['avg']+agg_stats['first_inference']['avg']) / 10 **6}
```

Comment:
Just for the record: absolute strange to divide by 1e3 or 1e6 depending on some key. This makes results from different runs hard to compare. Also use 1e3 and 1e6 as legit numbers, eliminating power operation every time

Reply:
I will leave as default time unit (microsends) to be consistent with the rest.